boards/adafruit_feather.json

```diff
@@ -0,0 +1,50 @@
+{
+  "build": {
+    "core": "earlephilhower",
+    "cpu": "cortex-m0plus",
+    "extra_flags": "-D ARDUINO_ADAFRUIT_FEATHER_RP2040 -DARDUINO_ARCH_RP2040 -DUSBD_MAX_POWER_MA=250",
+    "f_cpu": "133000000L",
+    "hwids": [
+      [
+        "0x2E8A",
+        "0x00C0"
+      ]
+    ],
+    "mcu": "rp2040",
+    "variant": "adafruit_feather",
+    "arduino": {
+      "earlephilhower": {
+        "boot2_source": "boot2_w25x10cl_4_padded_checksum.S",
+        "usb_vid": "0x239a",
+        "usb_pid": "0x80f1"
+      }
+    }
```

Comment:
Sorry I didn't mentioned it in the first review, could you please move the `arduino` section a bit higher, just above `core`? The fields in manifests are mostly alphabetically sorted, would be great to keep it that way.

Reply:
`arduino` attribute is now the first attribute in the `build` object.